<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvd::ColorSetupWidget</class>
 <widget class="QWidget" name="mvd::ColorSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="wHorizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="wLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>White (gray) channel.</string>
         </property>
         <property name="text">
          <string>&lt;span style=&quot;color:#EEEEEE;background-color:#111111&quot;&gt;&amp;nbsp;&lt;b&gt;W&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wLeftArrow">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;span&gt;&lt;b&gt;&amp;larr;&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="wComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>BAND 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="fileComponentLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Image bands&lt;br/&gt;â†“&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="rHorizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="rLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Red channel.</string>
         </property>
         <property name="text">
          <string>&lt;span style=&quot;color:#000000;background-color:#FF4444&quot;&gt;&amp;nbsp;&lt;b&gt;R&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rLeftArrow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;span&gt;&lt;b&gt;&amp;larr;&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="rComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>BAND 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="gHorizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="gLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Green channel.</string>
         </property>
         <property name="text">
          <string>&lt;span style=&quot;color:#000000;background-color:#44FF44&quot;&gt;&amp;nbsp;&lt;b&gt;G&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gLeftArrow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;span&gt;&lt;b&gt;&amp;larr;&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="gComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>BAND 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="bHorizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="bLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Blue channel.</string>
         </property>
         <property name="text">
          <string>&lt;span style=&quot;color:#CCCCCC;background-color:#4444FF&quot;&gt;&amp;nbsp;&lt;b&gt;B&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bLeftArrow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;span&gt;&lt;b&gt;&amp;larr;&lt;/b&gt;&amp;nbsp;&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="bComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>BAND 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAND 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="wCheckBox">
       <property name="toolTip">
        <string>Enable/disable gray-scale mode.</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="alphaLabel">
       <property name="text">
        <string>Alpha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="alphaSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="pageStep">
        <number>500</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rComboBox</tabstop>
  <tabstop>gComboBox</tabstop>
  <tabstop>bComboBox</tabstop>
  <tabstop>wCheckBox</tabstop>
  <tabstop>wComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>wComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>wLeftArrow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>wLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>bLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>bLeftArrow</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>bComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gLeftArrow</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rLeftArrow</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
